.github/workflows/container-tests.yml

```diff
@@ -0,0 +1,90 @@
+            context: "CentOS7"
+            compose: "CentOS-7"
+          - tmt_plan: "rhel7-docker"
+            os_test: "rhel7"
+            context: "RHEL7"
+            compose: "RHEL-7.9-Released"
+          - tmt_plan: "rhel8-docker"
+            os_test: "rhel8"
+            context: "RHEL8"
+            compose: "RHEL-8.3.1-Released"
+          - tmt_plan: "c9s"
+            os_test: "c9s"
+            context: "CentOS Stream 9"
+            compose: "CentOS-Stream-9"
+
+    if: |
+      github.event.issue.pull_request
+      && (contains(github.event.comment.body, '[test]') || contains(github.event.comment.body, '[test-all]'))
+      && contains(fromJson('["OWNER", "MEMBER"]'), github.event.comment.author_association)
+    steps:
+      - name: Get pull request number
+        id: pr_nr
+        run: |
+          PR_URL="${{ github.event.comment.issue_url }}"
+          echo "::set-output name=PR_NR::${PR_URL##*/}"
+
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          ref: "refs/pull/${{ steps.pr_nr.outputs.PR_NR }}/head"
+
+      - name: SHA value
+        id: sha_value
+        run: |
+          echo "::set-output name=SHA::$(git rev-parse HEAD)"
+
+      - name: Setup Testing Farm values
+        id: tf_values
+        run: |
+          # Update ubuntu-20.04 in order to install curl and jq
+          sudo apt update && sudo apt -y install curl jq
+          branch_name="master"
+          api_key="${{ secrets.TF_INTERNAL_API_KEY }}"
+          tmt_repo="https://gitlab.cee.redhat.com/platform-eng-core-services/sclorg-tmt-plans"
+          if [ "${{ matrix.tmt_plan }}" == "fedora" ] || [ "${{ matrix.tmt_plan }}" == "centos7" ] || [ "${{ matrix.tmt_plan }}" == "c9s" ]; then
+            api_key="${{ secrets.TF_PUBLIC_API_KEY }}"
+            branch_name="main"
+            tmt_repo="https://github.com/sclorg/sclorg-testing-farm"
+          fi
+          echo "::set-output name=API_KEY::$api_key"
+          echo "::set-output name=BRANCH_NAME::$branch_name"
+          echo "::set-output name=TMT_REPO::$tmt_repo"
+        shell: bash
+
+      # https://github.com/sclorg/testing-farm-as-github-action
+      - name: Schedule tests on external Testing Farm by Testing-Farm-as-github-action
+        id: github_action
+        uses: sclorg/testing-farm-as-github-action@v1
+        with:
+          api_key: ${{ steps.tf_values.outputs.API_KEY }}
+          git_url: ${{ steps.tf_values.outputs.TMT_REPO }}
+          git_ref: ${{ steps.tf_values.outputs.BRANCH_NAME }}
+          tmt_plan_regex: ${{ matrix.tmt_plan }}
+          pull_request_status_name: ${{ matrix.context }}
+          variables: "REPO_URL=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY;REPO_NAME=$GITHUB_REPOSITORY;PR_NUMBER=${{ steps.pr_nr.outputs.PR_NR }};OS=${{ matrix.os_test }};TEST_NAME=test"
+          secrets: "QUAY_USERNAME=${{ secrets.QUAY_IMAGE_SCLORG_BUILDER_USERNAME }};QUAY_TOKEN=${{ secrets.QUAY_IMAGE_SCLORG_BUILDER_TOKEN }}"
```

Comment:
Doesn't Fedora require `QUAY_IMAGE_BUILDER_{USERNAME,TOKEN}` instead of the sclorg one?

Reply:
Good point. Only `c9s` is valid. I would leave it as it is. RHEL7 and RHEL8 do not need it as well. But two the same actions, does not make sense

Reply:
And what will be the effect on Fedora tests, if we leave it as it is?

Reply:
Nothing, the secrets are hidden. Only c9s.yml use it https://github.com/sclorg/sclorg-testing-farm/blob/main/plans/c9s.fmf#L22

Reply:
I would rather doublecheck if [this line](https://github.com/sclorg/sclorg-testing-farm/blob/c2778a0a8fb2ea9120576c3cccde76892e389779/plans/c9s.fmf#L22) is really needed in the tmt plan before merging this PR.

Reply:
It is not needed at all. I will remove it.

Reply:
https://github.com/sclorg/sclorg-testing-farm/pull/13

Reply:
Thanks.